.eslintrc

```diff
@@ -1,12 +1,18 @@
 {
   "extends": "airbnb-base",
+  "parser": "babel-eslint",
   "rules": {
       "no-console": 0,
       "consistent-return": 0,
       "func-names": 0,
       "padded-blocks": 0,
       "import/no-extraneous-dependencies":0,
       "no-unused-vars": 0
-  }
+  },
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true
+}
```

Comment:
Can you just fix the indentation here and add a new line at the end of  the file?